File: tests/data/function/func.cpp
double simple(double a = 1, bool b = true) {
  return b ? a : 2;
}
std::function<std::string(std::string a, std::string b)> func = [](std::string a, std::string b) -> std::string {
  return a + b;
};
std::function<void()> arrow = []() -> void {};

File: tests/data-conversion/function/func.ts
function Simple(i: number) {
  return i;
}
function TakeCallback(input: number, callback: (i: number) => number) {
  return () => callback(input);
}
function TestLocalFunction() {
  let add = function(a: number) { return a + 1 };
  let arrow = () => {};
  Simple(1234);
  add(8963);
  arrow();
  const passLambda = TakeCallback(1234, add);
  const passFunction = TakeCallback(1234, Simple);
}
